src/pages/shin/Login/JeounghoLogin.js

```diff
@@ -1,5 +1,74 @@
-import React from 'react';
-const JeounghoLogin = () => {
-  return <div>Login</div>;
-};
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './JeounghoLogin.scss';
+
+function JeounghoLogin() {
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [password, setPassword] = useState('');
+  const [validity, setValidity] = useState(true);
+
+  const changeId = e => {
+    setId(e.target.value);
+  };
+
+  const changePassword = e => {
+    setPassword(e.target.value);
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+  };
+
+  const onKeyUp = validity => {
+    return id.includes('@') && password.length >= 5
+      ? setValidity(!validity)
+      : setValidity(validity);
+  };
+
+  const handleClick = () => {
+    return (
+      id.includes('@') &&
+      password.length >= 5(alert('로그인 성공'), navigate('/jeoungho-Main'))
+    );
+  };
+
+  return (
+    <main className="login">
```

Comment:
```suggestion
    <main className="jeounghoLogin">
```
scss 를 사용할 때, 가장 부모가 되는 태그의 className에는 컴포넌트의 이름이 동일하게 들어가는 것이
wecode 컨벤션입니다!